### PR TITLE
Added function for checking if value is inside a certain range.

### PR DESCRIPTION
**Pull request by emilwestergren**
@marcusnaslund peer review

### CODE REVIEW

**Comment by marcusnaslund** [comment]
It's really not clear that this method should expect milliseconds. Another idea might be two other `TimeSpan` instances.

And if it should be numerical values, maybe of type `Long`, since that's what the `to...` methods return.

**Comment by emilwestergren** [comment]
Updated

File: source/base/TimeSpan.ooc [hunk 1]
```diff
@@ -21,6 +21,7 @@ TimeSpan: cover {
 	toHours: func -> Long { this ticks / DateTime ticksPerHour }
 	toDays: func -> Long { this ticks / DateTime ticksPerDay }
 	toWeeks: func -> Long { this ticks / DateTime ticksPerWeek }
+	inRange: func (start, end: This) -> Bool { this >= start && this <= end }
```
Comment (marcusnaslund) [inline source/base/TimeSpan.ooc]: Yes, this makes more sense.

File: test/base/TimeSpanTest.ooc [hunk 2]
```diff
@@ -128,6 +128,13 @@ TimeSpanTest: class extends Fixture {
 			expect(Bstring == "0 0 text 0 60")
 			(defaultString, dayString, Astring, Bstring) free()
 		})
+		this add("inRange", func {
+			span := TimeSpan milliseconds(30)
+			expect(!span inRange(TimeSpan milliseconds(1), TimeSpan milliseconds(25)))
+			expect(!span inRange(TimeSpan milliseconds(31), TimeSpan milliseconds(50)))
```
Comment (marcusnaslund) [inline test/base/TimeSpanTest.ooc]: `expect(span inRange(TimeSpan milliseconds(31), TimeSpan milliseconds(50)), is false)` and similarly on the line above

**Comment by emilwestergren** [comment]
updated

File: source/base/TimeSpan.ooc (hunk 1 above)
Comment (marcusnaslund) [inline source/base/TimeSpan.ooc]: We do have the similar `in` defined on numeric types (`6 in(2 .. 8) == true`) but using a keyword as method name isn't good practice, methinks. So if you're happy with `inRange` I won't bother you about it.

Comment (emilwestergren) [inline source/base/TimeSpan.ooc]: This is fine. I will use it for sanity checking metadata, and almost all of them are `TimeSpan`

**Pull request merged**